docs/Migrate to v11.md

```diff
@@ -0,0 +1,215 @@
+
+```dart
+mapboxMap.style.setStyleURI(MapboxStyles.STANDARD);
+```
+
+The Mapbox Standard style features 4 light presets: `day`, `dusk`, `dawn`, and `night`. The style light preset can be changed from the default, `day`, to another preset with a single line of code. Here you identify which imported style (`basemap`) you want to change the `lightPresent` config on, as well as the value (`dusk`) you want to change it to.
+
+
+```dart
+mapboxMap.style.setStyleImportConfigProperty("basemap", "lightPreset", "dusk");
+```
+
+Changing the light preset will alter the colors and shadows on your map to reflect the time of day. For more information, check out the New 3D Lighting API section.
+
+Similarly, you can set other configuration properties on the Standard style such as showing POIs, place labels, or specific fonts:
+
+```dart
+mapboxMap.style.setStyleImportConfigProperty("basemap", "showPointOfInterestLabels", false);
+```
+
+The Standard style offers 6 configuration properties for developers to change when they import it into their own style:
+
+Property | Type | Description
+--- | --- | ---
+`showPlaceLabels` | `Bool` | Shows and hides place label layers.
+`showRoadLabels` | `Bool` | Shows and hides all road labels, including road shields.
+`showPointOfInterestLabels` | `Bool` | Shows or hides all POI icons and text.
+`showTransitLabels` | `Bool` | Shows or hides all transit icons and text.
+`lightPreset` | `String` | Switches between 4 time-of-day states: `dusk`, `dawn`, `day`, and `night`.
+`font` | `List` | Defines font family for the style from predefined options.
+
+Mapbox Standard is making adding your own data layers easier for you through the concept of `slot`s. `Slot`s are pre-specified locations in the style where your layer will be added to (such as on top of existing land layers, but below all labels). To do this, we've added a new `slot` property to each `Layer`. This property allows you to identify which `slot` in the Mapbox Standard your new layer should be placed in. To add custom layers in the appropriate location in the Standard style layer stack, we added 3 carefully designed slots that you can leverage to place your layer. These slots will remain stable, so you can be sure that your own map won't break even as the basemap evolves automatically.
+
+Slot | Description
+--- | ---
+`bottom` | Above polygons (land, landuse, water, etc.)
+`middle` | Above lines (roads, etc.) and behind 3D buildings
+`top` | Above POI labels and behind Place and Transit labels
+not specified | Above all existing layers in the style
+
+```dart
+final layer = LineLayer(id: "line-layer", sourceId: "line-source", slot: "middle");
+mapboxMap.style.addLayer(layer);
+```
+
+- Important: For the new Standard style, you can only add layers to these three slots (`bottom`, `middle`, `top`) within the Standard style basemap.
```

Comment:
GitHub has a special syntax for important notes:

> [!IMPORTANT]
> https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

I'm not 100% sure if it will work in the markdown documents